build.gradle.kts

```diff
@@ -59,7 +59,7 @@ tasks {
     // Configure the plugin's compatibility range with IntelliJ IDEA builds.
     patchPluginXml {
         sinceBuild.set("222")
-        untilBuild.set("233.*")
+        untilBuild.set("241.*")
```

Comment:
if possible, let's update also line 70 to use a latest version: 
`version.set("2024.1.4")`

Reply:
Updated the target version